app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -263,19 +264,41 @@ class AdvancedFilter extends React.Component {
   apply = () => {
     this.setState({ show: false, applied: true }, () => {
       this.props.advancedFilterUpdate(this.state.activeFilterOptions);
+      if (this.props.updateStickySettings) {
+        localStorage.setItem(`SaraFilter`, this.state.activeFilter.id);
```

Comment:
This throws an error if `activeFIlter` is nil.

To replicate just try and apply a filter.

1.

Reply:
good catch - added a check